src/components/inputs/rich-text-input/rich-text-input.story.js

```diff
@@ -36,6 +37,8 @@ storiesOf('Components|Inputs', module)
               false
             )}
             value={value}
+            onBlur={action('onBlur')}
```

Comment:
Do we need these here?

Reply:
Yeah, it's nice to have them in the story for debugging. We have the same in a bunch of other components.